grape/app/vertex_data_context.h

```diff
@@ -0,0 +1,53 @@
+/** Copyright 2020 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef GRAPE_APP_VERTEX_DATA_CONTEXT_H_
+#define GRAPE_APP_VERTEX_DATA_CONTEXT_H_
+#include "grape/grape.h"
```

Comment:
grape/grape.h is a header for examples,
include headers what you used.

Reply:
fixed